demo/project_demo12-HopsworksTikTokRecSys/tiktok-gen/src/interactions.rs

```diff
@@ -1,12 +1,9 @@
-use chrono::{DateTime, Datelike, Days, TimeDelta, Utc};
+use chrono::{DateTime, Datelike, TimeDelta, Utc};
```

Comment:
can't we reuse the existing datagen?

Reply:
We can, I just simplified it more to match the java version.

Reply:
I think it's a good idea to also have a datagen configuration for this benchmark, in addition to this custom generator, even if it's not precisely identical to Java. Can be a separate PR.

@abhizer, you can ask @gz to help tune that.

Reply:
Oh by datagen, you mean the new feldera internal generator?

Reply:
yes, it feeds data directly to the pipeline, so the pipeline is no bottlenecked by Kafka.

Reply:
if we can use datagen for the benchmark it will also shave off 15mins of runtime so that would be nice